linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/configure_opensc_card_drivers/rule.yml

```diff
@@ -31,7 +31,7 @@ identifiers:
 
 references:
     disa: 765,766,767,768,771,772,884
-    nist: IA-2(2)
+    nist: IA-2(2),IA-2(4)
```

Comment:
Should this be IA-2(1), IA-2(2), IA-2(3), and IA-2(4)?

ref: https://nvd.nist.gov/800-53/Rev4/control/IA-2